sdk/batch/Azure.Compute.Batch/samples/Sample1_CreatePool_Job_Task.md

```diff
@@ -0,0 +1,96 @@
+# Creating a Batch Pool, Job, and Tasks
+
+This sample demonstrates how to create a pool, job, and tasks for a Batch account.
+
+## Special note for Batch pools
+
+Both the **Azure.Compute.Batch** and the ARM based **Azure.Management.Batch** sdk's support the creating and managing of Batch Pools with the only difference being that **Azure.Management.Batch** supports creating [Batch Pools with Managed Idenities](https://learn.microsoft.com/azure/batch/managed-identity-pools), for this reason creating pools via **Azure.Management.Batch** is considered a best practice.  This example will use **Azure.Management.Batch** for pool creation and thus will need to create an ArmClient to do so.
```

Comment:
```suggestion
Both the **Azure.Compute.Batch** and the ARM-based **Azure.ResourceManager.Batch** libraries support the creating and managing of Batch Pools with the only difference being that **Azure.Management.Batch** supports creating [Batch Pools with Managed Identities](https://learn.microsoft.com/azure/batch/managed-identity-pools), for this reason creating pools via **Azure.ResourceManager.Batch** is considered a best practice.  This example will use **Azure.ResourceManager.Batch** for pool creation and thus will need to create an `ArmClient` to do so.
```

nit: branding.   "Azure SDK" is the all-up product which contains a number of client and management libraries.  We've been asked not to use SDK to refer to an individual package.